apps/fulfillment-e2e/src/support/page_objects/picking.page.ts

```diff
@@ -0,0 +1,50 @@
+import { PickingProductFragment } from '../page_fragments/picking-product.fragment';
+import { PickingFragment } from '../page_fragments/picking.fragment';
+import { AbstractFAPage } from './abstract.page';
+
+export class PickingPage extends AbstractFAPage {
+  constructor(protected id: string) {
+    super();
+  }
+
+  pickingFragment = new PickingFragment();
+  pickingProductFragment = new PickingProductFragment();
+
+  url = `/picking-list/picking/${this.id}`;
+
+  getNotPickedTabId() {
+    return this.pickingFragment.getTabsList().eq(0);
+  }
```

Comment:
These methods should be replaced with getters like in https://github.com/spryker/oryx/pull/290/files#diff-9bd3a573ea4c23003a79ffaad57da245783f0963f4eac0bc56bbbc336603e9cbR14-R15